spec/did.md

```diff
@@ -0,0 +1,80 @@
+| `controller`  | String              | Yes      | Must be a URI. |
+| `publicKeyJwk`        | Object      | No       | Preferred. Represents a [JWK](https://www.w3.org/TR/did-core/#bib-rfc7517). Either this or `publicKeyMultibase` must be present. |
+| `publicKeyMultibase`  | String      | No       | Not preferred. Either this or `publicKeyJwk` must be present. |
+
+### Service Data Model
+
+Following from [this data model](https://www.w3.org/TR/did-core/#services).
+
+| Property          | JSON Representation | Required | Notes          |
+| ----------------- | ------------------- | -------- | -------------- |
+| `id`              | String              | Yes      | Recommended to be a fully qualified DID URI (e.g. `did:example:abcd#service-1`). |
+| `type`            | String              | Yes      | Must be a type defined in the [service registry](https://www.w3.org/TR/did-spec-registries/#service-types). |
+| `serviceEndpoint` | Array of Strings    | Yes      | String values must be URIs. |
+| `sig`             | Array of Strings    | No       | -                           |
+| `enc`             | Array of Strings    | No       | -                           |
+
+## DID Resolution Metadata Data Model
+
+DID Resolution Metadata is _always optional_.
+
+Following from [this data model](https://www.w3.org/TR/did-core/#did-resolution-metadata).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `contentType` | String              | No       | Media type of the DID Document. |
+| `error`       |  String             | No       | Required if there was an error during resolution. One of [`invalidDid`, `notFound`, `representationNotSupported`]. |
+
+
+## DID Document Metadata Data Model
+
+DID Document Metadata is _always optional_.
+
+Following from [this data model](https://www.w3.org/TR/did-core/#did-document-metadata).
+
+| Property   | JSON Representation  | Required | Notes          |
+| ---------- | -------------------- | -------- | -------------- |
+| `created`  | String               | No       | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value for when the DID was created. |
+| `updated`  | String               | No       | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value for when the DID was last updated. |
+| `deactivated` | Boolean           | No       | Required to be `true` if the DID is deactivated. |
+| `nextUpdate`  | String            | No       | [XML Datetime](https://www.w3.org/TR/xmlschema11-2/#dateTime) value for when the next update of the DID. |
+| `versionId`   | String            | No       | Represents the version of the last update operation. |
+| `nextversionId`| String           | No       | Represents the version of the next update operation. |
+| `equivalentId` | Array of Strings | No       | A stronger form of the `alsoKnownAs` property, guaranteed by the DID method. |
```

Comment:
what do you mean by `stronger form` here? 😃

Reply:
the language is taken directly from https://www.w3.org/TR/did-core/#did-document-metadata

open to suggestions to make it clearer

Reply:
maybe providing an example would help

Reply:
@decentralgabe Both Moe and I had included some form of an explanation for `equivalentId` in [`web5-go`](https://github.com/TBD54566975/web5-go/blob/256e91138ecc83a77e99c59a8ba957f27cbe30ac/dids/didcore/document.go#L245-L251) and [`web5-js`](https://github.com/TBD54566975/web5-js/blob/eba12e914f05ca1abcd43e2d64fdff124df1b461/packages/dids/src/types/did-core.ts#L266-L277), which are both based on the DID Core spec language.

Perhaps including the pre-registration / post-registration example that is referenced would be a happy medium that tries to provide more context without being overly verbose?

Reply:
Could also link directly to this note https://www.w3.org/TR/did-core/#h-note-10

Pre-registraion/post-registration as the example case is really good, but may be too wordy for this doc (not sure?)